pydantic/utils.py

```diff
@@ -149,7 +149,7 @@ def generate_model_signature(
 
     present_params = signature(init).parameters.values()
     merged_params: Dict[str, Parameter] = {}
-    var_kw = None
+    var_kw = Parameter('data', Parameter.VAR_KEYWORD, annotation=Any)
```

Comment:
My thoughts on this: https://github.com/samuelcolvin/pydantic/pull/1034#issuecomment-592161978